Sources/Extensions/SwiftStdlib/IntExtensions.swift

```diff
@@ -137,6 +137,14 @@ public extension Int {
 		}
 		return romanValue
 	}
+    
+    /// SwifterSwift: Rounds to the closest multiple of n
+    ///
+    ///        12.roundToNearest(5) -> 10
+    ///        59.roundToNearest(25) -> 50
```

Comment:
I'm not going to lie, code examples are really starting to bother me 😅 I'd much prefer a comment explaining that we use traditional grade school rounding and round up after 5

Reply:
Alright, I removed the examples